api/clients/airtable.ts

```diff
@@ -15,30 +15,29 @@ export class AirtableClient {
     this.base = new Airtable({ apiKey: apiKey }).base(baseId)
   }
 
-  async find<T>(tableName: string, id: string): Promise<T> {
-    const record = await this.base.table(tableName).find(id)
-    return record.fields as unknown as T
+  async find<T extends FieldSet>(tableName: string, id: string): Promise<T> {
```

Comment:
Without extending `FieldSet`, I was getting some type error in VSCode even though the type check script wasn't picking it up 🤷🏼‍♂️ 
Doesn't seem any harm in doing it.